source/deployment/guest-accounts.rst

```diff
@@ -0,0 +1,126 @@
+.. _guest-accounts:
+
+Guest Accounts (Beta) (E10 and E20)
+===================================
```

Comment:
Should we remove the (E10 and E20) from the header, as it's mentioned that it's available for "E10 and above" in the next line?

Reply:
I based it on the format used in "Custom Terms of Service" but am open to removing the redudancy.

Reply:
If there's an existing format, then definitely leave it as-is. :)